src/vs/platform/workspace/common/workspace.ts

```diff
@@ -106,8 +106,7 @@ export interface IWorkspace {
 	readonly configuration?: URI | null;
 }
 
-export interface IWorkspaceFolderData {
-
+export interface INamedFolder {
```

Comment:
I would rather not introduce a new interface here. It seems like it just exists because QueryBuilder isn't using index, seems simpler to work with `IWorkspaceFolderData`

Reply:
Done.